src/Microsoft.OData.Core/ODataBatchOperationResponseMessage.cs

```diff
@@ -94,6 +98,12 @@ public int StatusCode
         /// </summary>
         public IServiceProvider Container { get; private set; }
 
+        /// <summary>
+        /// For Json batch, this is the atomic group id of the corresponding request (null for top level request).
+        /// For multipart batch, this is not used and the value should be null.

```

Comment:
Technically, base class shouldn't comment on how one if its properties may/may not be used by derived class.  More importantly, as per previous comment, this should be set in multipart/mixed to the id used in the changeset boundary.

Reply:
Updated with general description.

---
In reply to: [156779179](https://github.com/OData/odata.net/pull/992#discussion_r156779179) [](ancestors = 156779179)